src/SDKs/ApiManagement/Management.ApiManagement/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 [assembly: AssemblyDescription("Provides Api management capabilities for Microsoft Azure.")]
 
 [assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.8.0.0")]
```

Comment:
Please update the same in the csproj and update packagereleasenotes

Reply:
updated